src/lib/components/customer/event/CategoryFilterBar.svelte

```
@@ -0,0 +1,74 @@
+
+  let { categories, activeCategory = $bindable() }: Props = $props();
+
+  function handleSelectCategory(value: string) {
+    activeCategory = value;
+    const url = new URL($page.url);
+    if (value) {
+      url.searchParams.set('category', value);
+    } else {
+      url.searchParams.delete('category');
+    }
+    goto(url.toString(), { keepFocus: true, noScroll: true });
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)